tools/sh/sh_configure.bzl

```diff
@@ -11,77 +11,136 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Configure the shell toolchain on the local machine."""
+"""Configure sh_toolchains based on the local machine."""
```

Comment:
@meteorcloudy Could you review the changes to `sh_toolchain` and the repo rule?

Reply:
Done!